client/src/assets/scss/overrides.scss

```diff
@@ -50,6 +50,7 @@
     font-weight: bold;
     color: $dk-text;
     letter-spacing: normal;
+    letter-spacing: 0;
```

Comment:
Why both?

Reply:
Because .... hmm.. i clearly don't know how to rebase 😆  
Will fix!